v2/cmd/zlint-gtld-update/main.go

```diff
@@ -108,7 +109,16 @@ var tldMap = map[string]GTLDPeriod{
 // getData fetches the response body bytes from an HTTP get to the provider url,
 // or returns an error.
 func getData(url string) ([]byte, error) {
-	resp, err := httpClient.Get(url)
```

Comment:
This was flagged by the `noctx` linter so I quickly reworked it to use a timeout.